### PR TITLE
[dagit] Tooltip -> Tooltip2

### PR DESCRIPTION
**Pull request by hellendag**
## Summary

Followup to #4561. Convert all `Tooltip` use cases to `Tooltip2`, add lint rule to enforce it.

## Test Plan

Buildkite. Sanity check behavior/rendering of tooltips in Dagit.

### CODE REVIEW

File: .buildkite/dagster-buildkite/dagster_buildkite/cli.py [hunk 1]
```diff
@@ -19,20 +21,26 @@ def dagster():
     if not dagit_only:
         all_steps += dagster_steps()
 
-        all_steps += [wait_step()]
-
-        if DO_COVERAGE:
-            all_steps += [coverage_step()]
-
-        # Trigger builds of the internal pipeline for builds on master
+        # Trigger builds of the internal pipeline.
```
Comment (bengotow) [inline .buildkite/dagster-buildkite/dagster_buildkite/cli.py:24]: It'd be nice to land this buildkite stuff as a separate commit just so it's more visible in the commit history maybe?

**Review by bengotow — COMMENTED** [review]
This looks good to me but it also seems like there's a lot of doc / python / buildkite stuff going on here that should be broken out of this PR

File: .buildkite/dagster-buildkite/dagster_buildkite/cli.py (hunk 1 above)
Comment (hellendag) [reply .buildkite/dagster-buildkite/dagster_buildkite/cli.py]: 🤔 I don't think this is in the PR...

Comment (hellendag) [reply .buildkite/dagster-buildkite/dagster_buildkite/cli.py]: Oh, it's probably because the base branch changed.
